frontend/src/components/NavBar.jsx

```diff
@@ -62,15 +67,18 @@ function NavBar() {
         </ul>
       </nav>
 
-      <BurgerMenu width={320}>
+      <BurgerMenu width={320} isOpen={isBurgerOpen} onStateChange={isMenuOpen}>
         <NavLink
           id="meetings"
           className="menu-item"
           activeClassName="bm-active"
           to="/campaigns"
           onClick={
```

Comment:
```suggestion
          onClick={() => {
            dispatch({ type: 'AMB_ALL' });
            dispatch({ type: 'ASS_ALL' });
            isBurgerOpenUpdate(false)
          }}
```

Reply:
This will change today, anyway thank you :D